src/ECER.Clients.RegistryPortal/ecer.clients.registryportal.client/src/components/inputs/EceEducation.vue

```diff
@@ -146,14 +146,7 @@
         <div v-if="previousNameRadio === 'other'">
           <v-row>
             <v-col>
-              <v-text-field
-                v-model="studentFirstName"
-                :rules="[Rules.required('Enter your first name')]"
-                label="First name on transcript"
-                variant="outlined"
-                color="primary"
-                maxlength="100"
-              ></v-text-field>
```

Comment:
We also need to change the entities on the backend. They have firstName as required. 

![image](https://github.com/user-attachments/assets/ac6c40b2-1f37-47dc-a1bc-d12e28a32d91)